<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alta</string>
  </property>
  <layout class="QFormLayout" name="Alta">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nombre</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="leNombre"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Apellido</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="leApellido"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Usuario</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="leUsuario"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Clave</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="leClave"/>
   </item>
   <item row="4" column="1">
    <widget class="MiLabel" name="lAlta">
     <property name="text">
      <string>Dar de alta</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MiLabel</class>
   <extends>QLabel</extends>
   <header>milabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
